pod2usage support

sasvirco opened the pull request:
Added pod2usage support

sasvirco committed:
POD

Added POD documentation, and pod2usage support

sasvirco committed:
remove USAGE

Removed the usage string as this is now covered by the pod2usage

kirichkov commented:
I'm not acquainted with pod2usage, but shouldn't the exit value also be specified?

sasvirco commented:
If neither the exit status nor the verbose level is specified, then the default is to use an exit status of 2 with a verbose level of 0.

kirichkov commented:
That's a problem. This would change the exit status, and scripts that depend on it. You should return exit status 1. Please fix the exit status and I'll merge it.

sasvirco committed:
Fix exitlevel to 1

Changed exitlevel to 1 from default 2 to satisfy dependencies with softwares that can't work with other non-zero exit codes.
